src/Draco.Lsp/Server/LanguageClientProxy.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Draco.Lsp.Server;
+
+/// <summary>
+/// Intercepts language client calls.
+/// </summary>
+internal class LanguageClientProxy : DispatchProxy
```

Comment:
```suggestion
// NOTE: Not sealed, because DispatchProxy generates a derived class from this at runtime.
/// <summary>
/// Intercepts language client calls.
/// </summary>
internal class LanguageClientProxy : DispatchProxy
```